sdk/keyvault/azure-security-keyvault-secrets/pom.xml

```diff
@@ -11,7 +11,7 @@
 
   <groupId>com.azure</groupId>
   <artifactId>azure-security-keyvault-secrets</artifactId>
-  <version>4.0.1</version> <!-- {x-version-update;com.azure:azure-security-keyvault-secrets;current} -->
+  <version>4.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-security-keyvault-secrets;current} -->
 
```

Comment:
why is the core dependency nit updated to the 1.2.0-beta.1 for SDK poms everywhere ?

Reply:
They are marked as using `dependency` instead of `current`. I only updated `current` to bump the versions of the pom files.

Reply:
In generally they shouldn't be updated automatically. We should make a decision to bump them if we want to rebaseline to the higher version of core.